logging/logger.go

```diff
@@ -0,0 +1,150 @@
+package logging
+
+import (
+	"github.com/pkg/errors"
+	"os"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	. "github.com/spdfg/elektron/logging/types"
+)
+
+// var config loggerConfig
+var formatter elektronFormatter
+var elektronLoggerInstance elektronLogger
+
+type elektronLogger interface {
+	setNext(next elektronLogger)
+	isEnabled() bool
```

Comment:
`isEnabled()` does not have to be part of the interface.

Reply:
Yes. Removed it from there